src/enzo/Grid.h

```diff
@@ -2091,6 +2091,23 @@ int zEulerSweep(int j, int NumberOfSubgrids, fluxes *SubgridFluxes[],
 				     FLOAT TestStarParticleStarVelocity[],
 				     FLOAT TestStarParticleStarPosition[]);
 
+/* Radiating Star Particle test: initialize particle */
+  int TestRadiatingStarParticleInitializeGrid(float TestStarParticleStarMass, 
+				     float *Initialdt,
+				     FLOAT TestStarParticleStarVelocity[],
+				     FLOAT TestStarParticleStarPosition[],
+                 float UniformDensity,
+               float UniformTotalEnergy,
+               float UniformVelocity[],
+               float UniformBField[],
+               float InitialTemperature,
+               float PhotonTestInitialFractionHII, 
```

Comment:
Change all of the `PhotonTest` variables to start with `TestStarParticle` to be consistent with other parameters.